config/project.cmake

```diff
@@ -38,11 +38,12 @@ else()
   message(STATUS "Note: using 32 bit integer ids.")
 endif()
 
+cinch_load_extras()
 #------------------------------------------------------------------------------#
 # Add library targets
 #------------------------------------------------------------------------------#
 
-cinch_add_library_target(flecsi-sp specializations)
+cinch_add_library_target(flecsi-sp flecsi-sp)
```

Comment:
I'm not sure what effect this will have on projects that use `flecsi-sp` as an installed library though...